intelmq/bots/collectors/misp/collector.py

```diff
@@ -46,29 +47,31 @@ def process(self):
 
         # Process the response and events
         if 'response' in misp_result:
-
-            # Extract the MISP event details
-            for e in misp_result['response']:
-                misp_event = e['Event']
-
-                # Send the results to the parser
-                report = self.new_report()
-                report.add('raw', json.dumps(misp_event, sort_keys=True))
-                report.add('feed.url', self.parameters.misp_url)
-                self.send_message(report)
-
-            # Finally, update the tags on the MISP events.
-            # Note PyMISP does not currently support this so we use
-            # the API URLs directly with the requests module.
-
-            for misp_event in misp_result['response']:
-                # Remove the 'to be processed' tag
-                self.misp.remove_tag(misp_event,
-                                     self.parameters.misp_tag_to_process)
-
-                # Add a 'processed' tag to the event
-                self.misp.add_tag(misp_event,
-                                  self.parameters.misp_tag_processed)
+            # Old response content
```

Comment:
By "old" you mean the old version of pymisp?

Reply:
yes, older than 2019